frontend/test/metabase/scenarios/question/settings.cy.spec.js

```diff
@@ -138,6 +138,10 @@ describe("scenarios > question > settings", () => {
         cy.icon("play")
           .last()
           .click();
+
+        // Prevent performing actions while the query is being executed.
+        // Which caused some race condition and failed the test.
+        cy.findByText(/Doing science/).should("not.exist");
```

Comment:
This is the culprit for [all](https://app.circleci.com/pipelines/github/metabase/metabase/32363/workflows/2df69eb8-cd75-4ffc-a3e3-5dc276f64d59/jobs/1654770) [the](https://app.circleci.com/pipelines/github/metabase/metabase/32363/workflows/2df69eb8-cd75-4ffc-a3e3-5dc276f64d59/jobs/1654805) [failed](https://app.circleci.com/pipelines/github/metabase/metabase/32130/workflows/db05248f-0cb7-480b-9591-770e9f871e4a/jobs/1639793) [tests](https://app.circleci.com/pipelines/github/metabase/metabase/32122/workflows/ad5b2fff-02af-43d9-86f9-ac80f4934535/jobs/1639214).

Basically, it's a race condition where Cypress removes the field before the query is finished executing, causing the `card` to be overridden by an older version (before removing the field).

Reply:
@nemanjaglumac Seems like tests in settings.cy.spec.js could use a little race-condition tweaking by waiting for requests.
Also note the fail on Electron and not on Chrome, which we talked about earlier this week.